### PR TITLE
Add test to cover binding using annotations on CRD

### PR DESCRIPTION
**Pull request by baijum**
### Motivation

Fixes https://issues.redhat.com/browse/APPSVC-681

### Changes

Add test case

### Testing

```
TEST_ACCEPTANCE_TAGS=@wip make test-acceptance VERBOSE=3
```
For further more details refer the CONTRIBUTING.md

### CODE REVIEW

File: test/acceptance/features/bindAppToService.feature [hunk 1]
```diff
@@ -531,3 +531,57 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-etcd" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-etcd" should be changed to "True"
         And Application endpoint "/api/todos" is available
+
+    Scenario: Bind values from service CRD annotation
```
Comment (Avni-Sharma) [inline test/acceptance/features/bindAppToService.feature:535]: Can we/ should we change `bind` to `export` ?

Comment (baijum) [reply test/acceptance/features/bindAppToService.feature:535]: I just followed the previous test scenarios in this file. 
`Bind an imported Node.js application to Etcd database`
`Bind an imported Node.js application to PostgreSQL database in the following order: Service Binding, DB and Application`

Comment (baijum) [reply test/acceptance/features/bindAppToService.feature:535]: I am going for this: `Provide binding info through backing service CRD annotation`

**Pull request merged**